packages/mobile/src/screens/settings-screen/AccountSettingsScreen.tsx

```diff
@@ -45,10 +45,9 @@ const messages = {
   passwordTitle: 'Change Password',
   passwordDescription: 'Change your password',
   passwordButtonTitle: 'Change',
-  deactivateAccountTitle: 'Deactivate Account',
-  deactivateAccountDescription:
-    'Deactivate your account. This cannot be undone',
-  deactivateAccountButtonTitle: 'Deactivate',
+  deactivateAccountTitle: 'Delete Account',
```

Comment:
Nbd but could change the variable name too

Reply:
I opted not to because the codebase is litered with refs to "deactivate." this way the change is only scoped to "copy"